ext/tags_test.go

```diff
@@ -68,18 +66,10 @@ func (n noopTracer) StartSpanWithOptions(opts opentracing.StartSpanOptions) open
 	return noopSpan{Tags: make(opentracing.Tags)}
 }
 
-func (n noopTracer) Extractor(format interface{}) opentracing.Extractor {
-	return noopInjectorExtractor{}
-}
-
-func (n noopTracer) Injector(format interface{}) opentracing.Injector {
-	return noopInjectorExtractor{}
-}
-
-func (n noopInjectorExtractor) InjectSpan(span opentracing.Span, carrier interface{}) error {
-	return nil
+func (n noopTracer) Inject(toInject opentracing.Span, format interface{}, carrier interface{}) error {
+	panic("not implemented")
```

Comment:
I think the idea here was that `tags_test` should never be calling these methods so the panic's help reinforce that understanding. In any case, I was just trying to make Inject symmetrical with https://github.com/opentracing/opentracing-go/pull/65/files#diff-9f1aa8a53149ad3fc609b2518bf25139L84